rdflib/plugins/sparql/evaluate.py

```diff
@@ -357,10 +357,10 @@ def _buildQueryStringForServiceCall(ctx, match):
             service_query = "BASE <" + base + "> " + service_query
     sol = ctx.solution()
     if len(sol) > 0:
-        variables = " ".join(map(lambda v: v.n3(), sol))
-        variables_bound = " ".join(map(lambda v: ctx.get(v).n3(), sol))
+        variables = ' '.join([v.n3() for v in sol])
+        variables_bound = ' '.join([ctx.get(v).n3() for v in sol])
```

Comment:
Is this fully equivalent? As of https://docs.python.org/3.8/library/functions.html#map map operated in parallel, does `[v.n3() for v in sol]` also run in parallel?